pkg/server/image_pull.go

```diff
@@ -17,14 +17,315 @@ limitations under the License.
 package server
 
 import (
-	"errors"
+	gocontext "context"
+	"encoding/json"
+	"fmt"
+	"github.com/golang/glog"
+	"path/filepath"
+	"sync"
+	"time"
 
+	"github.com/containerd/containerd/content"
+	containerd_images "github.com/containerd/containerd/images"
+	"github.com/containerd/containerd/remotes"
+	"github.com/containerd/containerd/remotes/docker"
+	"github.com/kubernetes-incubator/cri-containerd/pkg/metadata"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"golang.org/x/net/context"
-
+	"golang.org/x/sync/errgroup"
```

Comment:
Add space before `k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime`, we should group different imports together.